### ISSUE TITLE
GraphQL relationship with RDS Aurora 

### ISSUE THREAD

**Issue by danielphung90** [aws-amplify/amplify-cli#1561]
>**Note**: If your question is regarding the AWS Amplify Console service, please log it in the 
[official AWS Amplify Console forum](https://forums.aws.amazon.com/forum.jspa?forumID=314&start=0)

** Which Category is your question related to? **

** What AWS Services are you utilizing? **

** Provide additional details e.g. code snippets **

**Comment by kaustavghosh06** [comment]
@danielphung90 You can modify the annotated graphql schema located in `amplify/backend/api<api-name>/schema.graphql` to link the types.

**Comment by pmd30011991** [comment]
Yes I did modify in schema.graphql do I need to manually create the resolver files? For each linkage?

**Comment by danielphung90** [comment]
> @danielphung90 You can modify the annotated graphql schema located in `amplify/backend/api<api-name>/schema.graphql` to link the types.

yeah which I did. but my main concern is: is there anyway to generate the resolvers and stacks base on schema.graphql. because after I generate the schema by using `amplify api add-graphql-datasource` then all the relationship become `id: Int!` what I want to do is link them without any change at resolver and stack part

> Yes I did modify in schema.graphql do I need to manually create the resolver files? For each linkage?

**Comment by ashwindevendran** [comment]
@danielphung90  Currently there is no support for automatic generation of the RDS resolvers after the schema has been edited with relationships (I presume this is what you mean by linking types to one another). You can however manually create your own resolvers to handle relationships.

In the meantime we do recognize the benefits of generating the resolvers to handle relationship and so we will treat this as a feature request.

**Comment by jukkamarttinen** [comment]
What is the current status of this request?

**Comment by cookmscott** [comment]
@ashwindevendran, I've been scouring the internet looking for details of how to map connections for RDS in GraphQL + AppSync. I've added some [feedback|https://github.com/aws-amplify/docs/issues/2431] for the documentation which does not state that table relationships are not generated by the cli, even though the example includes customers with orders. It is rather confusing, and tbh what's the point in having a relational datastore if the relationships aren't honored in GraphQL! I think it would be AMAZING if the cli handled the relationships out of the box... because it's not clear how to set them up manually.
